### PR TITLE
Include "3.x" tag in 3.12 release post

### PR DESCRIPTION
**Pull request by chriskrycho**
## What it does
Makes this entry show up in the **Browse by Major Version** list.

### CODE REVIEW

File: source/2019-08-16-ember-3-12-released.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Ember 3.12 Released
 author: Kenneth Larsen
-tags: Releases, 2019, 3, 3.12
+tags: Releases, 2019, 3, 3.12, 3.x
```
Comment (mansona) [inline source/2019-08-16-ember-3-12-released.md]: It looks like the other releases the tag is actually `Version 3.x` 🤔 for example https://raw.githubusercontent.com/ember-learn/ember-blog/master/source/2019-07-15-ember-3-11-released.md

Can you verify before we merge?

**Review by mansona — CHANGES_REQUESTED** [review]
I'm not entirely sure that's the right tag 🤔 have you been able to run this locally or is it just a text edit?

File: source/2019-08-16-ember-3-12-released.md (hunk 1 above)
Comment (chriskrycho) [reply source/2019-08-16-ember-3-12-released.md]: Doh, good point; I failed at reading. Updating!

**Comment by chriskrycho** [comment]
@mansona good catch; this was a text edit via GitHub.

**Review by mansona — APPROVED** [review]

**Pull request merged**